### PR TITLE
Add followup changelog message for #5278

### LINKED ISSUE #8555: RuntimeError: no running event loop
### Describe the bug

It looks like a breaking change in the latest version. 

In version 3.9.5 it works:

```
import aiohttp
connector = aiohttp.TCPConnector()
```

(in a more complex example I get a warning (DeprecationWarning: The object should be created within an async function))

but in 3.10 I get an exception:

```
Traceback (most recent call last):
  File "/tmp/a1.py", line 5, in <module>
    connector = aiohttp.TCPConnector()
  File "/home/tomek/work/github/kubernetes-client/kubernetes_asyncio-new/VENV/lib/python3.10/site-packages/aiohttp/connector.py", line 784, in __init__
    super().__init__(
  File "/home/tomek/work/github/kubernetes-client/kubernetes_asyncio-new/VENV/lib/python3.10/site-packages/aiohttp/connector.py", line 234, in __init__
    loop = loop or asyncio.get_running_loop()
RuntimeError: no running event loop
```

### To Reproduce

code
```
import aiohttp
connector = aiohttp.TCPConnector()
```

### Expected behavior

If the change is intentional it would be very useful to describe that in a changelog. 
If not I expect the same behavior like in the previous version.

### Logs/tracebacks

```python-traceback
Traceback (most recent call last):
  File "/tmp/a1.py", line 5, in <module>
    connector = aiohttp.TCPConnector()
  File "/home/tomek/work/github/kubernetes-client/kubernetes_asyncio-new/VENV/lib/python3.10/site-packages/aiohttp/connector.py", line 784, in __init__
    super().__init__(
  File "/home/tomek/work/github/kubernetes-client/kubernetes_asyncio-new/VENV/lib/python3.10/site-packages/aiohttp/connector.py", line 234, in __init__
    loop = loop or asyncio.get_running_loop()
RuntimeError: no running event loop
```
```


### Python Version

```console
$ python --version
3.10
```

### aiohttp Version

```console
$ python -m pip show aiohttp
Name: aiohttp
Version: 3.10.0
```

### multidict Version

```console
$ python -m pip show multidict
Name: multidict
Version: 6.0.4
```

### yarl Version

```console
$ python -m pip show yarl
Name: yarl
Version: 1.8.2
```

### OS

Linux

### Related component

Client

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow the aio-libs Code of Conduct

### PR DESCRIPTION
**Pull request by bdraco**
closes #8555

## What do these changes do?

Improve the changelog documentation for #5278

#5278 removed the deprecation warning about having a running event loop and replaced it with 
`asyncio.get_running_loop()` which will raise `RuntimeError` if there is no running loop.

Since the deprecation was done in #3372, which first appeared in https://github.com/aio-libs/aiohttp/releases/tag/v3.5.0 (~5 years, 7 months ago), reverting and extending the deprecation period did not seem warranted as five years should be enough notice.

### CODE REVIEW

File: CHANGES/8555.breaking.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Creating :class:`aiohttp.TCPConnector`, :class:`aiohttp.CookieJar`,
+ or :class:`aiohttp.Client` instances without a running event loop
```
Comment (bdraco) [inline CHANGES/8555.breaking.rst]:
```suggestion
Creating :class:`aiohttp.TCPConnector` or :class:`aiohttp.CookieJar`,
 instances without a running event loop
```

File: CHANGES/8555.breaking.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Creating :class:`aiohttp.TCPConnector` or :class:`aiohttp.CookieJar`,
+ instances without a running event loop
+ was deprecated in :issue:`3372`. After :issue:`5278`, attempting
+ attempting to do so will raise a :exc:`RuntimeError`.
```
Comment (bdraco) [inline CHANGES/8555.breaking.rst]:
```suggestion
 instances without a running event loop was deprecated
 in :issue:`3372`. After :issue:`5278`, attempting
 to do so will raise a :exc:`RuntimeError`.
```

File: CHANGES/8555.breaking.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Creating :class:`aiohttp.TCPConnector` or :class:`aiohttp.CookieJar`,
```
Comment (bdraco) [inline CHANGES/8555.breaking.rst]:
```suggestion
Creating :class:`aiohttp.TCPConnector` or :class:`aiohttp.CookieJar`
```

File: CHANGES/8555.breaking.rst [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Creating :class:`aiohttp.TCPConnector` or :class:`aiohttp.CookieJar`
+ instances without a running event loop now raises a :exc:`RuntimeError`.
```
Comment (webknjaz) [inline CHANGES/8555.breaking.rst]:
```suggestion
 instances without a running event loop now raises a :exc:`RuntimeError`.

```

**Comment by bdraco** [comment]
This likely affects ClientSession as well so we should add that to the list

**Pull request merged**